sklearn/multioutput.py

```diff
@@ -648,6 +648,51 @@ def _log_message(self, *, estimator_idx, n_estimators, processing_msg):
             return None
         return f"({estimator_idx} of {n_estimators}) {processing_msg}"
 
+    def _get_chain_method(self):
+        # Different chain methods allowed in `ClassifierChain` only
+        try:
+            chain_method = self.chain_method_
+        except AttributeError:
+            chain_method = "predict"
+        return chain_method
+
+    def _get_Y_output(self, X, *, output_method="predict"):
```

Comment:
maybe `_get_predictions` would be a nicer name.
I would not give a default for `output_method` and keep it as a keyword only argument to make it explicit in the code.

Reply:
Done!